receiver/gitmetricsreceiver/README.md

```diff
@@ -0,0 +1,34 @@
+# Git Metrics Receiver
+
+<!-- status autogenerated section -->
+| Status        |           |
+| ------------- |-----------|
+| Stability     | [development]: metrics   |
+| Distributions | [] |
+
+[development]: https://github.com/open-telemetry/opentelemetry-collector#development
+<!-- end autogenerated section --> 
+
+The Git Metrics receiver generates metrics from Git about Git repositories and 
```

Comment:
```suggestion
The Git Metrics receiver generates metrics about Git repositories and 
```

The receiver generates metrics not from Git itself, but from various vendor endpoints, as you correctly describe in the end of this sentence. 🙂